lib/pbench/server/api/resources/users_api.py

```diff
@@ -308,13 +307,12 @@ def __init__(self, logger, auth):
 
     def get_valid_target_user(
         self, target_username: str, request_type: str
-    ) -> "TargetUser":
+    ) -> TargetUser:
         """
         Helper function to determine whether the API call is permitted for the target username
-        Right now it is only permited for an admin user and the target user itself
-        :param target_username: str
-        :param request_type: str
-        :return: NamedTuple containing target User object/None, http_status_to_return, http_message_to_return
+        Right now it is only permited for an admin user and the target user itself.
+        This returns a target User on success or None on failure; in the case of failure,
+        also returns the corresponding HTTP status code and corresponding message string
```

Comment:
You could streamline this a bit to "returns the corresponding HTTP status code and message string".